editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -150,24 +151,32 @@ class SnapDialog : public ConfirmationDialog {
 		grid_step_y->set_select_all_on_focus(true);
 		child_container->add_child(grid_step_y);
 
-		child_container = memnew(GridContainer);
-		child_container->set_columns(2);
-		container->add_child(child_container);
-
 		label = memnew(Label);
 		label->set_text(TTR("Primary Line Every:"));
 		label->set_h_size_flags(Control::SIZE_EXPAND_FILL);
 		child_container->add_child(label);
 
-		primary_grid_steps = memnew(SpinBox);
-		primary_grid_steps->set_min(0);
-		primary_grid_steps->set_step(1);
-		primary_grid_steps->set_max(100);
-		primary_grid_steps->set_allow_greater(true);
-		primary_grid_steps->set_suffix(TTR("steps"));
-		primary_grid_steps->set_h_size_flags(Control::SIZE_EXPAND_FILL);
-		primary_grid_steps->set_select_all_on_focus(true);
-		child_container->add_child(primary_grid_steps);
+		primary_grid_step_x = memnew(SpinBox);
+		primary_grid_step_x->set_min(1);
+		primary_grid_step_x->set_step(1);
+		primary_grid_step_x->set_max(100);
+		primary_grid_step_x->set_max(SPIN_BOX_GRID_RANGE);
```

Comment:
This is set twice.

Reply:
Whoops! Would you prefer the arbitrary value of 100, or `SPIN_BOX_GRID_RANGE`?

Reply:
I think you can use `SPIN_BOX_GRID_RANGE` here :slightly_smiling_face: